SimCalorimetry/HcalTrigPrimAlgos/src/HcalTriggerPrimitiveAlgo.cc

```diff
@@ -516,13 +516,12 @@ void HcalTriggerPrimitiveAlgo::analyzeQIE11(IntegerCaloSamples& samples,
       int sampleTSminus1 = samples[ibin];
 
       if (fix_saturation_ && (sample_saturation.size() > ibin + 1))
-        check_sat = (sample_saturation[ibin + 1] || (sampleTS >= QIE11_MAX_LINEARIZATION_ET) ||
-                     sample_saturation[ibin] || (sampleTSminus1 >= QIE11_MAX_LINEARIZATION_ET));
+        check_sat |=  sample_saturation[ibin + 1] | (sampleTS > QIE11_MAX_LINEARIZATION_ET));
```

Comment:
I think there is one closing parenthesys too much (I did not test my suggestion, sorry for that... I did not test this one too then please check :-) )
```suggestion
        check_sat |=  sample_saturation[ibin + 1] | sampleTS > QIE11_MAX_LINEARIZATION_ET;
```
(perhaps without parenthesys is harder to fail)

Reply:
Ah, indeed, fixed in a9f9680